app/bundles/PluginBundle/Assets/js/plugin.js

```diff
@@ -306,4 +305,24 @@ Mautic.getIntegrationCampaigns = function (el, settings) {
             Mautic.removeLabelLoadingIndicator();
         }
     );
+};
+Mautic.pushConnectwiseActivities = function (el) {
```

Comment:
This is interfering with the toggle onchange methods. It's ok to not clear these out if we're checking for the value of `push_activities` in ConnectwiseIntegration before we push the activity.